src/authorization/AuthorizationsDefinition.ts

```diff
@@ -48,8 +61,19 @@ const AUTHORIZATION_DEFINITION: AuthorizationDefinition = {
           'createdBy.name', 'createdBy.firstName', 'lastChangedBy.name', 'lastChangedBy.firstName'
         ]
       },
-      { resource: Entity.TAGS, action: [Action.LIST, Action.IMPORT, Action.EXPORT], attributes: ['*'] },
-      { resource: Entity.TAG, action: [Action.CREATE, Action.UPDATE, Action.DELETE, Action.READ], attributes: ['*'] },
+      {
+        resource: Entity.TAGS, action: Action.LIST, attributes: [
+          'id', 'userID', 'active', 'ocpiToken', 'description', 'issuer', 'default',
+          'createdOn', 'lastChangedOn'
+        ]
+      },
+      { resource: Entity.TAGS, action: [Action.IMPORT, Action.EXPORT], attributes: ['*'] },
+      {
+        resource: Entity.TAG, action: Action.READ, attributes: [
+          'id', 'userID', 'issuer', 'active', 'description', 'default', 'user.id', 'user.name', 'user.firstName', 'user.email'
```

Comment:
As @jerome-benoit mentioned, we have a new field visualID in tags that should be added in the authorisations. 
Thanks

Reply:
Adding it now. 👍